models/Asset/Image/Thumbnail/Processor.php

```diff
@@ -171,7 +163,7 @@ public static function process(Asset $asset, Config $config, $fileSystemPath = n
 
         $fileExtension = $format;
         if ($format == 'original') {
-            $fileExtension = \Pimcore\File::getFileExtension($fileSystemPath);
+            $fileExtension = $fileExt;
```

Comment:
`$fileSystemPath` was used here as well (pushed that directly to your branch), but we can use directly `$fileExt` instead, so that's fine 😊

Reply:
Missed that one 😲
Thanks!!